test/metabase/driver/sql_jdbc_test.clj

```diff
@@ -17,7 +17,7 @@
 (set! *warn-on-reflection* true)
 
 (deftest ^:parallel describe-database-test
-  (is (= {:tables (set (for [table ["CATEGORIES" "VENUES" "CHECKINS" "USERS"]]
+  (is (= {:tables (set (for [table ["CATEGORIES" "VENUES" "CHECKINS" "USERS" "ORDERS" "PEOPLE" "PRODUCTS" "REVIEWS"]]
```

Comment:
Now that `sample-dataset` is moved into `test-data`, `test-data` has 8 tables, not 4.